yt_dlp/extractor/facebook.py

```diff
@@ -495,11 +498,15 @@ def extract_metadata(webpage):
             description = description or self._html_search_meta(
                 ['description', 'og:description', 'twitter:description'],
                 webpage, 'description', default=None)
+            uploader_data = (
```

Comment:
why was this moved

Reply:
first, 'or get_first(post, ('event', 'event_creator', {dict})) or {})' added
second, move the 'uploader_data' block right before the 'uploader = ' seems more easier to read and maintain afterward

Reply:
Makes sense. Same as my other comment though, moving the whole code block made the diff harder to parse